docs/content/preview/yugabyte-cloud/managed-automation/managed-guide-cli.md

```diff
@@ -0,0 +1,215 @@
+
+### Create and assign an IP allow list
+
+To connect to your cluster from your computer, you need an [IP allow list](../../cloud-secure-clusters/add-connections/) with your computer's IP address. Create one as follows:
+
+```sh
+ybm network-allow-list create \
+  --ip-addr $(curl ifconfig.me) \
+  --name my-computer
+```
+
+```output
+Name          Description   Allow List          Clusters
+my-computer                 170.200.10.100/32   
+NetworkAllowList my-computer successfully created
+```
+
+Assign the IP allow list to your Sandbox:
+
+```sh
+ybm cluster network allow-list assign \
+  --network-allow-list my-computer \
+  --cluster-name my-sandbox
+```
+
+```output
+The network allow list my-computer is being assigned to the cluster my-sandbox
+```
+
+To list your current network allow lists, enter the following command:
+
+```sh
+ybm network-allow-list list
+```
+
+```output
+Name                     Description              Allow List          Clusters
+my-computer                                       173.206.17.104/32   my-sandbox
+```
+
+### Download the cluster certificate
+
+To connect to a cluster in YugabyteDB Managed using a shell, you need the [cluster certificate](../../cloud-secure-clusters/cloud-authentication/). Download the certificate using the following command:
+
+```sh
+ybm cluster cert download --out $HOME/root.crt
+```
+
+### Show cluster settings
+
+To show details about your cluster, use the `describe` command as follows:
+
+```sh
+ybm cluster describe --cluster-name my-sandbox
+```
+
+```output
+General
+Name         ID                                     Version         State     Health
+my-sandbox   035f24da-3000-41c0-b12f-ef20004a3000   2.17.1.0-b439   ACTIVE    💚
+
+Provider   Tier      Fault Tolerance   Nodes     Total Res.(Vcpu/Mem/Disk)
+AWS        Sandbox   NONE              1         2 / 4GB / 10GB
+
+
+Regions
+Region      Nodes     vCPU/Node   Mem/Node   Disk/Node   VPC
+us-west-2   1         2           4GB        10GB        
+
+
+Endpoints
+Region      Accessibility   State     Host
+us-west-2   PUBLIC          ACTIVE    us-west-2.000f20da-3000-40c0-b10f-ef20004a3000.aws.ybdb.io
+
+
+Network AllowList
+Name          Description   Allow List
+my-computer                 173.200.10.100/32
+
+
+Nodes
+Name            Region[zone]            Health    Master    Tserver   ReadReplica   Used Memory(MB)
+my-sandbox-n1   us-west-2[us-west-2a]   💚        ✅        ✅        ❌            40MB
+```
+
+The host address to use to connect to your cluster is displayed under **Endpoints**.
+
+To connect to your cluster using the ysqlsh or ycqlsh shells, follow the instructions in [Connect via client shells](../../cloud-connect/connect-client-shell/). Use the cluster endpoint host address, the database credentials you specified when you created the sandbox, and the certificate you downloaded.
+
+## Create a single-region dedicated cluster
+
+The following command creates a single-region dedicated cluster in Tokyo:
```

Comment:
I know that I cant create more than one Sandbox cluster , so at this point in the tutorial , I would need some direction .. Either mention that the user must terminate their existing cluster to be able to create this one , or add a billing account to accommodate more.